README.md

```diff
@@ -1,2 +1,48 @@
 # Scripts
-Personal scripts used to automate various tasks
+
+Personal scripts used to automate various tasks.
+
+## Lab-grading
+
+Python script to automatically grade students.
+
+### Environment
```

Comment:
Move all the text below to its own `README.md` in the `lab-grading` folder and link the new `README.md` file here. This helps keep the global `README.md` as more scripts are added to this repo.